addon/utils/get-header.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+const { A, isNone } = Ember;
+
+/**
+ * Do a case-insensitive lookup of an HTTP header
+ *
+ * @function getHeader
+ * @private
+ * @param {Object} headers
+ * @param {string} name
+ * @return {string}
+ */
+export default function getHeader(headers, name) {
+  if (isNone(headers) || isNone(name)) {
+    return; // ask for nothing, get nothing.
+  }
+
+  let matchedKey = A(Object.keys(headers)).find((key) => {
```

Comment:
Fixed my additions, but after switching on `prefer-const`, ESLint is raising flags elsewhere in the code.  Mind if I do the config change + compliance fixes in a separate PR?